README.md

```diff
@@ -441,6 +442,10 @@ are responsible for reviewing and merging PRs as well as steering conversation a
 If you would like to become a maintainer, please review the Apache Airflow
 [committer requirements](https://cwiki.apache.org/confluence/display/AIRFLOW/Committers).
 
+## Can I use the Apache Airflow logo in my presentation?
+
+Yes! Be sure to abide by the Apache Foundation [trademark policies](https://www.apache.org/foundation/marks/pmcs#graphics). The most up to date logos are found in [the repo](/docs/img/logos)
```

Comment:
This may not be the best link as that page is mostly focused at the project itself. Let me see if I can find a better one.

Reply:
Something on https://www.apache.org/foundation/marks/ perhaps.

Reply:
https://www.apache.org/foundation/marks/#books more specifically?

Reply:
How about this one? http://www.apache.org/foundation/marks/guide#naming

Reply:
Oh perfect love the #books one. I'll make that change now.